packages/marshal/src/passStyleOf.js

```diff
@@ -194,8 +194,8 @@ export const passStyleOf = makePassStyleOf([
   CopyArrayHelper,
   CopyRecordHelper,
   TaggedHelper,
-  RemotableHelper,
   ErrorHelper,
+  RemotableHelper,
```

Comment:
Please also swap lines 11,12. Likewise lines 41,42. Just to maintain esthetic parallelism.

Reply:
Done.